WordPress/Sniffs/WP/DeprecatedParametersSniff.php

```diff
@@ -0,0 +1,317 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Check for usage of deprecated parameters in WP functions and provide alternative based on the parameter passed.
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.11.0
+ */
+class WordPress_Sniffs_WP_DeprecatedParametersSniff extends WordPress_AbstractFunctionParameterSniff {
+
+	/**
+	 * The group name for this group of functions.
+	 *
+	 * @since 0.11.0
+	 *
+	 * @var string
+	 */
+	protected $group_name = 'wp_deprecated_parameters';
+
+	/**
+	 * Minimum WordPress version.
+	 *
+	 * This sniff will throw an error when usage of deprecated parameters is
+	 * detected if the parameter was deprecated before the minimum supported
+	 * WP version; a warning otherwise.
```

Comment:
This first line seems more like a "long description" for the sniff itself, so might be better placed in the class doc.
Possibly the second line too.